lib/search_solr_tools/translators/nsidc_json.rb

```diff
@@ -75,9 +75,10 @@ def translate_sensor_to_facet_sensor(json)
         return facet_values if json.nil?
 
         json.each do |json_entry|
+          long_name = json_entry['shortName'].eql?(json_entry['longName']) ? '' : json_entry['longName']
           sensor_bin = Helpers::SolrFormat.facet_binning('sensor', json_entry['shortName'].to_s)
           facet_values << if sensor_bin.eql? json_entry['shortName']
-                            "#{json_entry['longName']} | #{json_entry['shortName']}"
+                            "#{long_name} | #{json_entry['shortName']}"
```

Comment:
If the long_name is blank (due to the change above) that would make this line look like ` | short_name`.  Is this the desired result?  Or would it be better to not have the divider in there?

Reply:
That is exactly the desired result. It's what the content looks like now.